lib/fedex/request/ground_close.rb

```diff
@@ -0,0 +1,58 @@
+require 'fedex/request/base'
+require 'fedex/ground_close'
+
+module Fedex
+  module Request
+    class GroundClose < Base
```

Comment:
this class is pretty weak on documentation for a contribution to a third party library